docs(operator): Update plugins config

aimeeu opened the pull request:
- add Gate proxy section
- add Spinnaker versions

aimeeu committed:
docs(operator): update plugins section

aimeeu committed:
Rearrange content on page

317brian reviewed (approved):
LGTM. Not sure if you need someone from the Extensibility team to look at it so not merging.

aimeeu committed:
Add more fields to yaml to make it placement clear

Rebase
Add more fields above the plugin config to make it clear that the
plugin stuff is at the same level as 'version'
Add  note about putting config in profiles: <service> section but
that this approach is for development.

aimeeu committed:
fix yaml spacing

aimeeu committed:
fix url link in alert.

aimeeu commented:
rolled into https://github.com/armory/docs/pull/174
